cmd/fyne/package.go

```diff
@@ -93,42 +93,75 @@ type packager struct {
 }
 
 func (p *packager) packageLinux() error {
-	prefixDir := ensureSubDir(ensureSubDir(p.dir, "usr"), "local")
-	shareDir := ensureSubDir(prefixDir, "share")
+	var prefixDir string
+	local := "local/"
+	tempDir := "tmp-pkg"
 
-	appsDir := ensureSubDir(shareDir, "applications")
-	desktop := filepath.Join(appsDir, p.name+".desktop")
-	deskFile, _ := os.Create(desktop)
-	_, err := io.WriteString(deskFile, "[Desktop Entry]\n"+
-		"Type=Application\n"+
-		"Name="+p.name+"\n"+
-		"Exec="+filepath.Base(p.exe)+"\n"+
-		"Icon="+p.name+"\n")
-	if err != nil {
-		return errors.Wrap(err, "Failed to write desktop entry string")
+	if p.install {
+		tempDir = ""
+	}
+
+	if _, err := os.Stat(filepath.Join("usr", "local")); os.IsNotExist(err) {
```

Comment:
This line does not work on my system - I guess it is rooted at PWD?
The suggestion, however, works.

```suggestion
	if _, err := os.Stat(filepath.Join("/", "usr", "local")); os.IsNotExist(err) {
```

Reply:
Yeah that true. Will get that fixes :)